packages/itwin/grouping-mapping-widget/src/components/Properties/GroupProperties/GroupPropertyUtils.ts

```diff
@@ -43,13 +43,12 @@ const convertType = (type: string): DataType => {
     case "int":
     case "enum":
     case "long":
+    case "number":
```

Comment:
I don't think the whole `convertType` functions is needed anymore after the removal of `propertyType` from `ECPropertyReference`.

Reply:
Does that mean we don't need to extract the propertyType from Primitives and PrimitiveStruct Properties?

Reply:
Resolved